tests/test_error.py

```diff
@@ -46,7 +46,7 @@ def test_missing_single_parameter_has_line_number(self):
             '    """This is missing.',
             '',
             '    Args:',
-            '        x: The first argument.',
+            '        x (int): The first argument.',
```

Comment:
If we disable this error by default (which we'll do), then these test edits shouldn't be necessary unless enabling the extension.